lib/msf/core/exploit/http/client.rb

```diff
@@ -37,6 +37,7 @@ def initialize(info = {})
           ]),
         OptString.new('HttpUsername', [false, 'The HTTP username to specify for authentication', '']),
         OptString.new('HttpPassword', [false, 'The HTTP password to specify for authentication', '']),
+        OptPath.new('HttpHeaders', [false, 'Path to raw HTTP headers to append to each request']),
```

Comment:
Should this be named `HttpRawHeaders` or `HttpAdditionalRawHeaders`?

I like the idea here, would like to be clear that there is no validation on header format just encase we want to add some option that injects validated headers down the road.

Reply:
Sure. I described that in the option description, but I can update the name, too.